### ISSUE TITLE
It should be possible to move a tree

### ISSUE THREAD

**Issue by rotsee** [fruktkartan/fruktkartan#28]
When in editing mode, it should be possible to move the tree.

**Comment by quite** [comment]
On the topic of editing existing trees... I'm actually thinking that the present possibility to change the type of a tree is more used for griefing (destructive editing), that legitimate use. I'm considering if we should even remove it. If somebody creates a new tree with the wrong type, it's nowadays quite easy for them to just mark it for deletion, and redo it. What do you think @rotsee

**Comment by rotsee** [comment]
Agreed! (I'm halfway through migrating the codebase to Vue3 but still not
finished, will fix in that codebase)
_______________
Leonard Wallentin
http://leowallentin.se

**Comment by quite** [comment]
Nice, looking forward to that rewrite! push a branch when you have something that at all builds and can be run?

I'm gonna disable editing of type in our current frontend as well
